types/aws-lambda/trigger/cognito-user-pool-trigger/define-auth-challenge.d.ts

```diff
@@ -12,7 +12,7 @@ export interface DefineAuthChallengeTriggerEvent extends BaseTriggerEvent<"Defin
         clientMetadata?: StringMap | undefined;
     };
     response: {
-        challengeName: string;
+        challengeName: string | undefined;
```

Comment:
Can you change this to `challengeName?: string | undefined` as per the rest of the file.